pkg/clustermanager/internal/deployments.go

```diff
@@ -1,13 +1,10 @@
 package internal
 
-import "github.com/aws/eks-anywhere/pkg/types"
-
 // map where key = namespace and value is a capi deployment
 var CAPIDeployments = map[string][]string{
 	"capi-kubeadm-bootstrap-system":     {"capi-kubeadm-bootstrap-controller-manager"},
 	"capi-kubeadm-control-plane-system": {"capi-kubeadm-control-plane-controller-manager"},
 	"capi-system":                       {"capi-controller-manager"},
-	"capi-webhook-system":               {"capi-controller-manager", "capi-kubeadm-bootstrap-controller-manager", "capi-kubeadm-control-plane-controller-manager"},
```

Comment:
Hmm, why is this removed?

Reply:
these changes are from the PR to start using v1beta1 bundle behind feature flag, commits from that PR are included in this one. In upstream v1.0.1 tags, webhook is deployed within the same pod and not a separate pod, hence there's no separate namespace/pod for webhooks.